src/components/button/button-test.stories.tsx

```diff
@@ -170,92 +188,100 @@ export const generateButtons = (buttonType, iconPosition) => {
                 )}
               </React.Fragment>
             ))}
-          </div>
+          </React.Fragment>
         );
       })}
-    </>
+    </Box>
   );
 };
 
-export const NoWrapButtonsStory = ({ noWrap }) => {
+export const NoWrapButtonsStory = ({ noWrap }: Partial<ButtonProps>) => {
   return (
-    <>
+    <Box>
       {BUTTON_VARIANTS.map((buttonType) => {
         return BUTTON_SIZES.map((size) => {
           return (
-            <div style={{ width: 100 }}>
-              <Button buttonType={buttonType} noWrap={noWrap} size={size}>
-                Long button text
-              </Button>
-              <Button
-                buttonType={buttonType}
-                noWrap={noWrap}
-                size={size}
-                iconType="bin"
-              >
-                Long button text
-              </Button>
-              <Button
-                buttonType={buttonType}
-                noWrap={noWrap}
-                size={size}
-                iconType="bin"
-                iconPosition="after"
-              >
-                Long button text
-              </Button>
-              <Button
-                buttonType={buttonType}
-                noWrap={noWrap}
-                size="large"
-                iconType="bin"
-                subtext="Even longer button subtext"
-              >
-                Long button text
-              </Button>
-              <Button
-                buttonType={buttonType}
-                noWrap={noWrap}
-                size="large"
-                iconType="bin"
-                iconPosition="after"
-                subtext="Even longer button subtext"
-              >
-                Long button text
-              </Button>
-            </div>
+            <React.Fragment key={`${buttonType}-${size}`}>
```

Comment:
I think that we do not need that `Fragment` here, as you are already wrapping that content in a `Box`